### PR TITLE
[UA] 20210311: Folk of the Feywild

### PR DESCRIPTION
**Pull request by NickVendel**
> Wander into the magical realm of the Feywild in this month’s Unearthed Arcana! With the playtest options in the PDF below, your D&D character can be a member of one of these new D&D races: fairy, hobgoblin of the Feywild, owlfolk, or rabbitfolk.

### CODE REVIEW

File: unearthed-arcana/2021/20210311.xml [hunk 1]
```diff
@@ -0,0 +1,382 @@
...
+			<p style="text-indent: -1em; margin-left: 1em"><strong>Hospitality.</strong> You and the target of your Help action each gain a number of temporary hit points equal to 1d6 plus your proficiency bonus.</p>
+			<p style="text-indent: -1em; margin-left: 1em"><strong>Passage.</strong> You and the target of your Help action each increase your walking speeds by 10 feet until the start of your next turn.</p>
+			<p style="text-indent: -1em; margin-left: 1em"><strong>Spite.</strong> Until the start of your next turn, the first time you or the target of your Help action hits a creature with an attack roll, that creature has disadvantage on the next attack roll that it makes within the next minute.</p>
+		</description>
+		<sheet action="Bonus Action" usage="{{proficiency}}/Long Rest">
+			<description>You can use this trait to take the Help action.</description>
+			<description level="3">You can use this trait to take the Help action. Additionally, choose one of the options below each time you take the Help action, whether as a bonus action or an action: &#13;
```
Comment (kgirtxd) [inline unearthed-arcana/2021/20210311.xml:195]: Maybe add bullet points to the Fey Gift upgrades to help them pop on the sheet.

File: unearthed-arcana/2021/20210311.xml [hunk 2]
```diff
@@ -0,0 +1,382 @@
...
+
+	<element name="Silent Feathers" type="Racial Trait" source="Unearthed Arcana: Folk of the Feywild" id="ID_WOTC_UA20210311_RACIAL_TRAIT_OWLFOLK_SILENT_FEATHERS">
+		<description>
+			<p>You have proficiency in the Stealth skill.</p>
+		</description>
+		<sheet display="false">
+			<description>You have proficiency in the Stealth skill.</description>
+		</sheet>
+		<rules>
+			<grant type="Proficiency" id="ID_PROFICIENCY_SKILL_STEALTH" />
```
Comment (kgirtxd) [inline unearthed-arcana/2021/20210311.xml:306]: Add TCOE Skill Swap option

File: unearthed-arcana/2021/20210311.xml [hunk 3]
```diff
@@ -0,0 +1,382 @@
...
+			<p class="indent"><strong><em>Nimble Flight.</em></strong> Thanks to your wings, you have a flying speed equal to your walking speed. When you fall, you can use your reaction to make a Dexterity saving throw (DC 10) to stop falling and fly in place until the start of your next turn.</p>
+			<p class="indent"><strong><em>Silent Feathers.</em></strong> You have proficiency in the Stealth skill.</p>
+		</description>
+		<sheet display="false" />
+		<rules>
+			<select type="Racial Trait" name="Size (Owlfolk)" supports="ID_INTERNAL_RACIAL_TRAIT_SMALL|ID_INTERNAL_RACIAL_TRAIT_MEDIUM" />
+			<stat name="innate speed" value="30" bonus="base" />
+			<select type="Ability Score Improvement" name="Ability Score Improvement +2 (Owlfolk)" supports="Custom Ability Score Increase 2" />
+			<select type="Ability Score Improvement" name="Ability Score Improvement +1 (Owlfolk)" supports="Custom Ability Score Increase 1" />
+			<grant type="Language" id="ID_LANGUAGE_COMMON" />
```
Comment (kgirtxd) [inline unearthed-arcana/2021/20210311.xml:233]: Add TCOE Language option

File: unearthed-arcana/2021/20210311.xml [hunk 4]
```diff
@@ -0,0 +1,382 @@
...
+			<p class="indent"><strong><em>Fortune from the Many.</em></strong> If you miss with an attack roll or fail an ability check or a saving throw, you can draw on your bonds of reciprocity to gain a bonus to the roll equal to the number of allies you can see within 30 feet of you (maximum bonus of +5). You can use this trait a number of times equal to your proficiency bonus, and you regain all expended uses you finish a long rest.</p>
+		</description>
+		<sheet display="false" />
+		<rules>
+			<grant type="Racial Trait" id="ID_INTERNAL_RACIAL_TRAIT_MEDIUM" />
+			<stat name="innate speed" value="30" bonus="base" />
+			<select type="Ability Score Improvement" name="Ability Score Improvement +2 (Hobgoblin)" supports="Custom Ability Score Increase 2" />
+			<select type="Ability Score Improvement" name="Ability Score Improvement +1 (Hobgoblin)" supports="Custom Ability Score Increase 1" />
+			<grant type="Language" id="ID_LANGUAGE_COMMON" />
```
Comment (kgirtxd) [inline unearthed-arcana/2021/20210311.xml:167]: Add TCOE Language option

File: unearthed-arcana/2021/20210311.xml [hunk 5]
```diff
@@ -0,0 +1,382 @@
...
+			<p class="indent"><strong><em>Lucky Footwork.</em></strong> When you fail a Dexterity saving throw, you can use your reaction to roll a d4 and add it to the result, potentially turning the failure into a success.</p>
+			<p class="indent"><strong><em>Rabbit Hop.</em></strong> Once during each of your turns when you walk at least 5 feet, you can hop, rolling a d12 and moving that many feet in a direction of your choice. This extra distance doesn’t cost movement, but you can hop only if your speed isn’t 0. </p>
+		</description>
+		<sheet display="false" />
+		<rules>
+			<select type="Racial Trait" name="Size (Rabbitfolk)" supports="ID_INTERNAL_RACIAL_TRAIT_SMALL|ID_INTERNAL_RACIAL_TRAIT_MEDIUM" />
+			<stat name="innate speed" value="30" bonus="base" />
+			<select type="Ability Score Improvement" name="Ability Score Improvement +2 (Rabbitfolk)" supports="Custom Ability Score Increase 2" />
+			<select type="Ability Score Improvement" name="Ability Score Improvement +1 (Rabbitfolk)" supports="Custom Ability Score Increase 1" />
+			<grant type="Language" id="ID_LANGUAGE_COMMON" />
```
Comment (kgirtxd) [inline unearthed-arcana/2021/20210311.xml:331]: Add TCOE Language option

File: unearthed-arcana/2021/20210311.xml [hunk 6]
```diff
@@ -0,0 +1,382 @@
...
+
+	<element name="Leporine Senses" type="Racial Trait" source="Unearthed Arcana: Folk of the Feywild" id="ID_WOTC_UA20210311_RACIAL_TRAIT_RABBITFOLK_LEPORINE_SENSES">
+		<description>
+			<p>You have proficiency in the Perception skill.</p>
+		</description>
+		<sheet display="false">
+			<description>You have proficiency in the Perception skill.</description>
+		</sheet>
+		<rules>
+			<grant type="Proficiency" id="ID_PROFICIENCY_SKILL_PERCEPTION" />
```
Comment (kgirtxd) [inline unearthed-arcana/2021/20210311.xml:360]: Add TCOE Skill Swap option

File: unearthed-arcana/2021/20210311.xml [hunk 7]
```diff
@@ -0,0 +1,382 @@
...
+			<p class="indent"><strong><em>Fey Passage.</em></strong> You can squeeze through a space as narrow as 1 inch wide.</p>
+		</description>
+		<sheet display="false" />
+		<rules>
+			<grant type="Racial Trait" id="ID_WOTC_UA20210311_RACIAL_TRAIT_FAIRY_TYPE_FEY" />
+			<grant type="Racial Trait" id="ID_INTERNAL_RACIAL_TRAIT_SMALL" />
+			<stat name="innate speed" value="30" bonus="base" />
+			<select type="Ability Score Improvement" name="Ability Score Improvement +2 (Fairy)" supports="Custom Ability Score Increase 2" />
+			<select type="Ability Score Improvement" name="Ability Score Improvement +1 (Fairy)" supports="Custom Ability Score Increase 1" />
+			<grant type="Language" id="ID_LANGUAGE_COMMON" />
```
Comment (kgirtxd) [inline unearthed-arcana/2021/20210311.xml:59]: Add TCOE Language option

**Review by kgirtxd — CHANGES_REQUESTED** [review]

File: unearthed-arcana/2021/20210126.xml [hunk 8]
```diff
@@ -187,7 +188,8 @@
 			<stat name="innate speed" value="30" bonus="base" />
 			<select type="Ability Score Improvement" name="Ability Score Improvement +2 (Hexblood)" supports="Custom Ability Score Increase 2" />
 			<select type="Ability Score Improvement" name="Ability Score Improvement +1 (Hexblood)" supports="Custom Ability Score Increase 1" />
-			<grant type="Language" id="ID_LANGUAGE_COMMON" />
+			<grant type="Language" id="ID_LANGUAGE_COMMON" requirements="!ID_WOTC_TCOE_OPTION_CUSTOMIZED_LANGUAGE" />
+			<select type="Language" name="Customized Language" supports="Custom Race Language" number="2"  requirements="ID_WOTC_TCOE_OPTION_CUSTOMIZED_LANGUAGE" />
```
Comment (kgirtxd) [inline unearthed-arcana/2021/20210126.xml:192]:
```suggestion
			<select type="Language" name="Customized Language" supports="Custom Race Language" requirements="ID_WOTC_TCOE_OPTION_CUSTOMIZED_LANGUAGE" />
```
Only Common is being swapped the other language has more options

File: unearthed-arcana/2021/20210126.xml [hunk 9]
```diff
@@ -89,7 +89,8 @@
 			<stat name="innate speed" value="35" bonus="base" />
 			<select type="Ability Score Improvement" name="Ability Score Improvement +2 (Dhampir)" supports="Custom Ability Score Increase 2" />
 			<select type="Ability Score Improvement" name="Ability Score Improvement +1 (Dhampir)" supports="Custom Ability Score Increase 1" />
-			<grant type="Language" id="ID_LANGUAGE_COMMON" />
+			<grant type="Language" id="ID_LANGUAGE_COMMON" requirements="!ID_WOTC_TCOE_OPTION_CUSTOMIZED_LANGUAGE" />
+			<select type="Language" name="Customized Language" supports="Custom Race Language" number="2"  requirements="ID_WOTC_TCOE_OPTION_CUSTOMIZED_LANGUAGE" />
```
Comment (kgirtxd) [inline unearthed-arcana/2021/20210126.xml:93]:
```suggestion
			<select type="Language" name="Customized Language" supports="Custom Race Language" requirements="ID_WOTC_TCOE_OPTION_CUSTOMIZED_LANGUAGE" />
```
Only Common is being swapped the other language has more options

File: unearthed-arcana/2021/20210126.xml [hunk 10]
```diff
@@ -334,7 +336,8 @@
 			<stat name="innate speed" value="30" bonus="base" />
 			<select type="Ability Score Improvement" name="Ability Score Improvement +2 (Reborn)" supports="Custom Ability Score Increase 2" />
 			<select type="Ability Score Improvement" name="Ability Score Improvement +1 (Reborn)" supports="Custom Ability Score Increase 1" />
-			<grant type="Language" id="ID_LANGUAGE_COMMON" />
+			<grant type="Language" id="ID_LANGUAGE_COMMON" requirements="!ID_WOTC_TCOE_OPTION_CUSTOMIZED_LANGUAGE" />
+			<select type="Language" name="Customized Language" supports="Custom Race Language" number="2"  requirements="ID_WOTC_TCOE_OPTION_CUSTOMIZED_LANGUAGE" />
```
Comment (kgirtxd) [inline unearthed-arcana/2021/20210126.xml:340]:
```suggestion
			<select type="Language" name="Customized Language" supports="Custom Race Language" requirements="ID_WOTC_TCOE_OPTION_CUSTOMIZED_LANGUAGE" />
```
Only Common is being swapped the other language has more options

File: unearthed-arcana/2021/20210311.xml [hunk 11]
```diff
@@ -0,0 +1,388 @@
...
+		</description>
+		<sheet display="false" />
+		<rules>
+			<grant type="Racial Trait" id="ID_WOTC_UA20210311_RACIAL_TRAIT_FAIRY_TYPE_FEY" />
+			<grant type="Racial Trait" id="ID_INTERNAL_RACIAL_TRAIT_SMALL" />
+			<stat name="innate speed" value="30" bonus="base" />
+			<select type="Ability Score Improvement" name="Ability Score Improvement +2 (Fairy)" supports="Custom Ability Score Increase 2" />
+			<select type="Ability Score Improvement" name="Ability Score Improvement +1 (Fairy)" supports="Custom Ability Score Increase 1" />
+			<grant type="Language" id="ID_LANGUAGE_COMMON" requirements="!ID_WOTC_TCOE_OPTION_CUSTOMIZED_LANGUAGE" />
+			<select type="Language" name="Customized Language" supports="Custom Race Language" number="2"  requirements="ID_WOTC_TCOE_OPTION_CUSTOMIZED_LANGUAGE" />
```
Comment (kgirtxd) [inline unearthed-arcana/2021/20210311.xml:60]:
```suggestion
			<select type="Language" name="Customized Language" supports="Custom Race Language" requirements="ID_WOTC_TCOE_OPTION_CUSTOMIZED_LANGUAGE" />
```
Only Common is being swapped the other language has more options

File: unearthed-arcana/2021/20210311.xml [hunk 12]
```diff
@@ -0,0 +1,388 @@
...
+			<p class="indent"><strong><em>Fortune from the Many.</em></strong> If you miss with an attack roll or fail an ability check or a saving throw, you can draw on your bonds of reciprocity to gain a bonus to the roll equal to the number of allies you can see within 30 feet of you (maximum bonus of +5). You can use this trait a number of times equal to your proficiency bonus, and you regain all expended uses you finish a long rest.</p>
+		</description>
+		<sheet display="false" />
+		<rules>
+			<grant type="Racial Trait" id="ID_INTERNAL_RACIAL_TRAIT_MEDIUM" />
+			<stat name="innate speed" value="30" bonus="base" />
+			<select type="Ability Score Improvement" name="Ability Score Improvement +2 (Hobgoblin)" supports="Custom Ability Score Increase 2" />
+			<select type="Ability Score Improvement" name="Ability Score Improvement +1 (Hobgoblin)" supports="Custom Ability Score Increase 1" />
+			<grant type="Language" id="ID_LANGUAGE_COMMON" requirements="!ID_WOTC_TCOE_OPTION_CUSTOMIZED_LANGUAGE" />
+			<select type="Language" name="Customized Language" supports="Custom Race Language" number="2"  requirements="ID_WOTC_TCOE_OPTION_CUSTOMIZED_LANGUAGE" />
```
Comment (kgirtxd) [inline unearthed-arcana/2021/20210311.xml:169]:
```suggestion
			<select type="Language" name="Customized Language" supports="Custom Race Language" requirements="ID_WOTC_TCOE_OPTION_CUSTOMIZED_LANGUAGE" />
```
Only Common is being swapped the other language has more options

File: unearthed-arcana/2021/20210311.xml [hunk 13]
```diff
@@ -0,0 +1,388 @@
...
+			<p class="indent"><strong><em>Silent Feathers.</em></strong> You have proficiency in the Stealth skill.</p>
+		</description>
+		<sheet display="false" />
+		<rules>
+			<select type="Racial Trait" name="Size (Owlfolk)" supports="ID_INTERNAL_RACIAL_TRAIT_SMALL|ID_INTERNAL_RACIAL_TRAIT_MEDIUM" />
+			<stat name="innate speed" value="30" bonus="base" />
+			<select type="Ability Score Improvement" name="Ability Score Improvement +2 (Owlfolk)" supports="Custom Ability Score Increase 2" />
+			<select type="Ability Score Improvement" name="Ability Score Improvement +1 (Owlfolk)" supports="Custom Ability Score Increase 1" />
+			<grant type="Language" id="ID_LANGUAGE_COMMON" requirements="!ID_WOTC_TCOE_OPTION_CUSTOMIZED_LANGUAGE" />
+			<select type="Language" name="Customized Language" supports="Custom Race Language" number="2"  requirements="ID_WOTC_TCOE_OPTION_CUSTOMIZED_LANGUAGE" />
```
Comment (kgirtxd) [inline unearthed-arcana/2021/20210311.xml:236]:
```suggestion
			<select type="Language" name="Customized Language" supports="Custom Race Language" requirements="ID_WOTC_TCOE_OPTION_CUSTOMIZED_LANGUAGE" />
```
Only Common is being swapped the other language has more options

File: unearthed-arcana/2021/20210311.xml [hunk 14]
```diff
@@ -0,0 +1,388 @@
...
+			<p class="indent"><strong><em>Rabbit Hop.</em></strong> Once during each of your turns when you walk at least 5 feet, you can hop, rolling a d12 and moving that many feet in a direction of your choice. This extra distance doesn’t cost movement, but you can hop only if your speed isn’t 0. </p>
+		</description>
+		<sheet display="false" />
+		<rules>
+			<select type="Racial Trait" name="Size (Rabbitfolk)" supports="ID_INTERNAL_RACIAL_TRAIT_SMALL|ID_INTERNAL_RACIAL_TRAIT_MEDIUM" />
+			<stat name="innate speed" value="30" bonus="base" />
+			<select type="Ability Score Improvement" name="Ability Score Improvement +2 (Rabbitfolk)" supports="Custom Ability Score Increase 2" />
+			<select type="Ability Score Improvement" name="Ability Score Improvement +1 (Rabbitfolk)" supports="Custom Ability Score Increase 1" />
+			<grant type="Language" id="ID_LANGUAGE_COMMON" requirements="!ID_WOTC_TCOE_OPTION_CUSTOMIZED_LANGUAGE" />
+			<select type="Language" name="Customized Language" supports="Custom Race Language" number="2"  requirements="ID_WOTC_TCOE_OPTION_CUSTOMIZED_LANGUAGE" />
```
Comment (kgirtxd) [inline unearthed-arcana/2021/20210311.xml:336]:
```suggestion
			<select type="Language" name="Customized Language" supports="Custom Race Language" requirements="ID_WOTC_TCOE_OPTION_CUSTOMIZED_LANGUAGE" />
```
Only Common is being swapped the other language has more options

**Review by kgirtxd — CHANGES_REQUESTED** [review]

**Review by kgirtxd — APPROVED** [review]

**Review by AngryGuineaPig — APPROVED** [review]

**Pull request merged**